src/snowflake/snowpark/_internal/telemetry.py

```diff
@@ -424,3 +428,17 @@ def send_large_query_breakdown_telemetry(
             },
         }
         self.send(message)
+
+    def send_large_query_optimization_skipped_telemetry(
```

Comment:
if we add a new compilation telementry file, will we be able to move all those definition to the new file?